### DISCUSSION TITLE
v3.2.109

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#14095] [General]
## What's Changed
* Enhanced default `focusedDate` logic in `DateTimePicker` by @smiliyas in https://github.com/filamentphp/filament/pull/14063
* Update header-cell.blade.php by @shopapps in https://github.com/filamentphp/filament/pull/14088

## New Contributors
* @shopapps made their first contribution in https://github.com/filamentphp/filament/pull/14088

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.108...v3.2.109

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.109'>v3.2.109</a>.</em>

**Comment by daom89** [comment]
#14063 Is generate error, 

```php
    public static function getDatePickerProgramacionFecha(): DatePicker
    {
        return DatePicker::make('fecha')
            ->label(__('general.labels.date'))
            ->required()
            //->rules(['required', 'date', self::getUniqueValidationRule()])
            ->timezone(config('app.timezone'))
            ->native(false)
            ->displayFormat('d/m/Y')
            ->format('Y-m-d')
            ->firstDayOfWeek(1)
            ->closeOnDateSelection()
            ->minDate(now()->subDays(1))
            ->maxDate(now()->addYear())
            ->disabledDates([

            ])
            ->suffixIcon('heroicon-o-calendar')
            ->prefixIcon('heroicon-m-calendar-days')
            ->prefixIconColor('primary')
            ->placeholder(__('general.placeholders.select_date'))
            ->weekStartsOnMonday()
            ->closeOnDateSelection()
            ->readonly(false);
    }
```
